src/m365/base/PowerBICommand.spec.ts

```diff
@@ -0,0 +1,27 @@
+import * as assert from 'assert';
+import { Logger } from '../../cli';
+import PowerBICommand from './PowerBICommand';
+
+class MockCommand extends PowerBICommand {
+  public get name(): string {
+    return 'mock';
+  }
+
+  public get description(): string {
+    return 'Mock command';
+  }
+
+  public commandAction(logger: Logger, args: any, cb: () => void): void {
+    cb();
+  }
+
+  public commandHelp(): void {
+  }
+}
+
```

Comment:
Maybe we can add some extra test cases around restoring auth and executing the command like it's done with the `GraphCommand`